sympy/geometry/ellipse.py

```diff
@@ -465,6 +465,50 @@ def evolute(self, x='x', y='y'):
         t2 = (self.vradius*(y - self.center.y))**Rational(2, 3)
         return t1 + t2 - (self.hradius**2 - self.vradius**2)**Rational(2, 3)
 
+    def director_circle(self, x='x', y='y'):
```

Comment:
Rather than this, let's look up how other places in SymPy handle this issue. I think a better way is for example this one:
```
def director_circle(self, x=None, y=None):
```
and if it's called as `director_circle()` then you use some Dummy variables. If it's called as `director_circle(a, b)` where `a` and `b` are symbols, then those would be used instead.

But there might be a more standard way to handle this in SymPy.

Reply:
what does it mean ``` def director_circle(self, x=None, y=None): ```

Reply:
I'll look it up & fix this as soon as possible @certik & @jksuom. Thanks for the reviews. Also, I need to add the test cases for this feature I created. How can I compare the equality of 2 expressions in Sympy. If my feature outputs for e.g. x**2 + y**2 - 25, how can assert that it equals x**2 + y**2 - 25 ?

Reply:
Also @certik , Does 
```
def director_circle(self, x=None, y=None):
```
have any advantages over 
```
def director_circle(self, x='x', y='y'):
```
Since I am still learning while contributing, It's best for me to get to know some good code practices which I can incorporate in other features which I am planning to incorporate in the geometry module.